presto-main/src/main/java/com/facebook/presto/util/GraphvizPrinter.java

```diff
@@ -325,6 +328,13 @@ public Void visitMarkDistinct(MarkDistinctNode node, Void context)
             return node.getSource().accept(this, context);
         }
 
+        @Override
+        public Void visitNativeExecution(NativeExecutionNode node, Void context)
+        {
+            printNode(node, "NativeExecution", NODE_COLORS.get(NodeType.NATIVE_EXECUTION));
+            return null;
```

Comment:
Should we recursively print all the nodes encapsulated by the NativeExecutionNode?
```
 node.getSubPlan().accept(this, context);
```